website/source/docs/platform/k8s/helm.html.md

```diff
@@ -340,6 +349,35 @@ to run the sync program.
       The name of the private key for the certificate file within the
       `secretName` secret.
 
+  * <a name="v-connectinject-acl-bindingrule-selector" href="#v-connectinject-acl-bindingrule-selector">`namespaceSelector`</a> (`string: "serviceaccount.name!=default"`) -
+  A [selector](/docs/acl/acl-auth-methods.html#binding-rules) for restricting automatic injection to only matching services based on
+  their associated service account. By default, services using the `default` Kubernetes service account will not have a proxy injected.
```

Comment:
```suggestion
 associated with a matching service account. By default, services using the `default` Kubernetes service account will not have a proxy injected.
```

Reply:
The way the feature was built, service names already have to match their service accounts for the Auth Method to work correctly, so this edit does not fully capture the functionality of this option. This selector allows you to include or exclude services that are associated with certain specific service accounts. One of the examples often used is to exclude any services using the `vault` service account by including a `serviceaccount.name!=vault` clause.